lighthouse-core/report/v2/renderer/details-renderer.js

```diff
@@ -165,15 +169,16 @@ class DetailsRenderer {
    * @param {!DetailsRenderer.ThumbnailDetails} value
    * @return {!Element}
    */
-  _renderThumbnail(value) {
-    if (/^image/.test(value.mimeType) === false) {
-      return this._dom.createElement('span');
-    }
+  _renderThumbnail(details) {
+    // TODO: use some empty details test instead? to handle data uris?
```

Comment:
yeah let's strip out all data URIs at the audit base-class, they can make the report super beefy anyway